app/post_order.py

```diff
@@ -0,0 +1,23 @@
+def postorder_traversal(root):
+    result = []
+    def traverse(node):
+        if node:
+            traverse(node.left)
+            traverse(node.right)
+            result.append(node.value)
+    traverse(root)
+    return result
+
+def is_prime(n):
```

Comment:
**Code optimization Suggestion**
```python
def is_prime(n):
    if n <= 1:
        return False
    if n <= 3:
        return True
    if n % 2 == 0 or n % 3 == 0:
        return False
    for i in range(5, int(n**0.5) + 1, 6):
        if n % i == 0 or n % (i + 2) == 0:
            return False
    return True
```
**Explanation for suggestion**
Replaced the while loop with a for loop to iterate over the range with a step of 6, which makes the code more concise and easier to understand. The upper limit of the range is the square root of n, which optimizes the function by reducing the number of iterations needed to check if n is prime.